docs/book/promotions.rst

```diff
@@ -4,37 +4,126 @@
 Promotions
 ==========
 
-...
+The system of **Promotions** in **Sylius** is really flexible. It is a combination of promotion rules and actions.
+
+Promotions are assigned to Orders. They have a few parameters - a unique ``code``, ``name``, ``usageLimit``,
+the period of time when it works.
+There is a possibility to define **exclusive promotions** (no other can be applied if an exclusive promotion was applied)
+and **priority** that is useful for them, because the exclusive promotion should get the top priority.
+
+How to create a Promotion programmatically?
+-------------------------------------------
+
+Just as usually, use a factory. The promotion needs a ``code`` and a ``name`` before it can be added to the repository.
+
+.. code-block:: php
+
+   /** @var PromotionInterface $promotion */
+   $promotion = $this->container->get('sylius.factory.promotion')->createNew();
+
+   $promotion->setCode('simple_promotion_1');
+   $promotion->setName('Simple Promotion');
+
+   $this->container->get('sylius.repository.promotion')->add($promotion);
```

Comment:
Would be better to create a simple promotion with action and then add it repo, this code does nothing for the shop currently. ;)